integration_tests/seeds/materialization/tst_fct_chained_views.csv

```diff
@@ -0,0 +1,14 @@
+parent_id,parent,child_id,child,distance
+model.dbt_project_evaluator_integration_tests.fct_model_9,fct_model_9,model.dbt_project_evaluator_integration_tests.stg_model_5,stg_model_5,1
+model.dbt_project_evaluator_integration_tests.int_model_4,int_model_4,model.dbt_project_evaluator_integration_tests.int_model_5,int_model_5,1
+model.dbt_project_evaluator_integration_tests.int_model_4,int_model_4,model.dbt_project_evaluator_integration_tests.dim_model_7,dim_model_7,2
+model.dbt_project_evaluator_integration_tests.int_model_5,int_model_5,model.dbt_project_evaluator_integration_tests.dim_model_7,dim_model_7,1
+model.dbt_project_evaluator_integration_tests.stg_model_1,stg_model_1,model.dbt_project_evaluator_integration_tests.int_model_4,int_model_4,1
+model.dbt_project_evaluator_integration_tests.stg_model_1,stg_model_1,model.dbt_project_evaluator_integration_tests.int_model_5,int_model_5,1
+model.dbt_project_evaluator_integration_tests.stg_model_1,stg_model_1,model.dbt_project_evaluator_integration_tests.int_model_5,int_model_5,2
+model.dbt_project_evaluator_integration_tests.stg_model_1,stg_model_1,model.dbt_project_evaluator_integration_tests.dim_model_7,dim_model_7,2
+model.dbt_project_evaluator_integration_tests.stg_model_1,stg_model_1,model.dbt_project_evaluator_integration_tests.dim_model_7,dim_model_7,3
+model.dbt_project_evaluator_integration_tests.stg_model_2,stg_model_2,model.dbt_project_evaluator_integration_tests.stg_model_4,stg_model_4,1
+model.dbt_project_evaluator_integration_tests.stg_model_2,stg_model_2,model.dbt_project_evaluator_integration_tests.dim_model_7,dim_model_7,2
+model.dbt_project_evaluator_integration_tests.stg_model_3,stg_model_3,model.dbt_project_evaluator_integration_tests.fct_model_6,fct_model_6,1
```

Comment:
i think this is a view --> table relationship. Is this intended to be included?

Reply:
This is intentional yes, because my original idea was to show what model is getting impacted by the chain of views. In that case, `fct_model_6` is potentially impacted by the fact that there are `x` views upstream

Reply:
Changing the name of the fact table to make it clearer!